src/content/eth2/get-involved/medalla-data-challenge/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Medalla data challenge
+description: Instructions, deadlines, and prize information for the Eth2 Medalla testnet data challenge
+lang: en
+sidebar: true
+---
+
+# The Eth2 Medalla Data Challenge
+
+_Up until 20 October 2020, the Ethereum Foundation is sponsoring a Medalla data analysis and data visualization blog post challenge. Here are all the details you need._
+
+<Divider />
+
+Calling data scientists, data engineers, data visualizers, developers, and anyone interested in digging into Ethereum data!
+
+The Ethereum 2.0 [Medalla testnet](https://github.com/goerli/medalla) is live. Validators are validating. Attesters are attesting. Blocks are propagating. Which means, data, lots and lots of data. And **the Ethereum community needs your help to make sense of it all.**
+
+- _What data visualizations can show the health of the Medalla test network?_
+- _Aggregation performance. Latencies. Validator effectiveness. Gini coefficients. Block rewards. What compelling story can you tell with Medalla data?_
+- _What performance and behavior patterns can you find among different classes of validators (for example hobbyists, institutional players, exchanges, and stakers)?_
+- _What patterns exist in slashings?_
+- _What other data can you collect and analyze on Medalla?_
+- _What new tools can you build to collect and analyze data on Medalla?_
+
+<InfoBanner emoji=":thinking_face:">
+    Wait, how do I get started with Medalla? See <a href="/en/eth2/get-involved/#medalla">how to stake on the Medalla testnet</a>
+ </InfoBanner>
+
+## The challenge
+
+**Document your best Medalla data insights in the most readable blog post possible – for prizes!**
+
+We're running this challenge because there's a lot to learn and discover from Medalla testnet activity. Your findings will give the Ethereum community —from beginners to core researchers— important insight into Medalla, the last step en route to Phase 0, the [Beacon Chain](/en/Eth2/the-beacon-chain/).
```

Comment:
```suggestion
We're running this challenge because there's a lot to learn and discover from Medalla testnet activity. Your findings will give the Ethereum community —from beginners to core researchers— important insight into Medalla, the last step en route to Phase 0, the [Beacon Chain](/en/eth2/the-beacon-chain/).
```